src/basic/mountpoint-util.c

```diff
@@ -446,14 +446,15 @@ bool fstype_needs_quota(const char *fstype) {
 }
 
 bool fstype_is_api_vfs(const char *fstype) {
-        const FilesystemSet *fs;
+        assert(fstype);
 
-        FOREACH_POINTER(fs,
-                filesystem_sets + FILESYSTEM_SET_BASIC_API,
-                filesystem_sets + FILESYSTEM_SET_AUXILIARY_API,
-                filesystem_sets + FILESYSTEM_SET_PRIVILEGED_API,
-                filesystem_sets + FILESYSTEM_SET_TEMPORARY)
-            if (nulstr_contains(fs->value, fstype))
+        const FilesystemSet *fs;
+        VA_ARGS_FOREACH(fs,
```

Comment:
Isn't this a bit confusing? All these uses are not in `va` functions, no?

Reply:
`VA_ARGS_FOREACH` just goes through all passed arguments. Existing uses in codebase don't involve `va_arg` either.

Maybe, rename this to `FOREACH_ARGUMENTS`?

Reply:
Sure, that works, or simply provide both as aliases